### PR TITLE
cleanup ci.rake tasks

### PR DESCRIPTION
**Pull request by peetucket**
remove unnecessary tasks; update readme; add default rake task for running tests; add rake task for rebuild cassettes

### CODE REVIEW

**Comment by drh-stanford** [comment]
I rebased on master to get #53 and pushed up. Passes Travis now.

File: lib/tasks/ci.rake [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
-desc 'Run continuous integration suite (assuming jetty is not yet started)'
+desc 'Run continuous integration suite (tests, coverage, rubocop, docs)'
 task :ci do
-  Rake::Task['rubocop'].invoke
   unless Rails.env.test?
     system('bundle exec rake ci RAILS_ENV=test')
   else
+    system('bundle exec rspec spec --color')
+    Rake::Task['rubocop'].invoke
+    system('yard doc')
```
Comment (drh-stanford) [inline lib/tasks/ci.rake]: can we omit building the documentation in the ci build? i don't think the output docs are used by anything.

Comment (peetucket) [inline lib/tasks/ci.rake]: sure.. just did it for completeness but easy enough to omit

File: lib/tasks/ci.rake [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
 require 'rest_client'
 
-desc 'Run continuous integration suite (assuming jetty is not yet started)'
+desc 'Run continuous integration suite (tests, coverage, rubocop, docs)'
 task :ci do
-  Rake::Task['rubocop'].invoke
   unless Rails.env.test?
     system('bundle exec rake ci RAILS_ENV=test')
   else
+    system('bundle exec rspec spec --color')
```
Comment (drh-stanford) [inline lib/tasks/ci.rake]: we already have a `.rspec` file for configuration so this can just be `bundle exec rspec` i think

Comment (peetucket) [inline lib/tasks/ci.rake]: and possibly just Rake::Task['rspec']

Comment (peetucket) [inline lib/tasks/ci.rake]: i'm not sure you need to explicitly run rails_env=test either

**Pull request merged**